app/uk/gov/hmrc/helptosavefrontend/views/register/account_created.scala.html

```diff
@@ -20,7 +20,7 @@
 @import uk.gov.hmrc.helptosavefrontend.views.html.helpers.ga_submit_button
 @import uk.gov.hmrc.helptosavefrontend.config.FrontendAppConfig
 
-@(accountNumber: String)(implicit htsContext: HtsContext, request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+@(accountNumber: String, email: String, endOfMonthDate: String)(implicit htsContext: HtsContext, request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
```

Comment:
it'd be more exact for the type of `endOfMonthDate` to be `LocalDate` rather than `String` - the FE view is then in charge of how to format that date to display

Reply:
but also, the end of month date shouldn't be included as part of this ticket - (see the updated comments on the ticket), so this needs to be taken out anyway for now

Reply:
ok this has been removed for now